Tests/MapboxMapsTests/Location/Pucks/Puck2DIntegrationTests.swift

```diff
@@ -133,6 +133,60 @@ class Puck2DIntegrationTests: MapViewIntegrationTestCase {
         wait(for: [addedPrecisePuckExpectation, removedPrecisePuckExpectation, addedApproximatePuckExpectation], timeout: 5)
     }
 
+    func testAccuracyRadiusIsHidden() throws {
+        let style = try XCTUnwrap(self.style)
+        style.uri = .streets
+
+        let location = Location(with: CLLocation(latitude: 1, longitude: 1))
+        let accuracyRingIsHiddenExpectation = XCTestExpectation(description: "Layer does not contain an accuracy ring")
+
+        didFinishLoadingStyle = { _ in
+            let puck = Puck2D(puckStyle: .precise,
+                              puckBearingSource: .heading,
+                              locationSupportableMapView: self.mapView!,
+                              style: style,
+                              configuration: Puck2DConfiguration())
+            do {
+                try puck.createPreciseLocationIndicatorLayer(location: location)
+            } catch {
+                XCTFail("Failed to create a precise location indicator layer.")
+            }
+
+            let layer = try! style.layer(withId: "puck") as LocationIndicatorLayer
+            if layer.accuracyRadius == nil {
+                accuracyRingIsHiddenExpectation.fulfill()
+            }
+        }
+        wait(for: [accuracyRingIsHiddenExpectation], timeout: 5)
```

Comment:
we should update this test so that the only time it would ever take the full 5 seconds is if the style loaded callback is not invoked. this means that we need to fulfill the expectation unconditionally if the callback is invoked.

Reply:
this will help to keep our test suite running quickly in more cases

Reply:
I'm not sure I fully follow what the solution would be here. The conditional is needed to verify that there is/isn't accuracy ring properties present